pkg/cmd/pr/checkout/checkout_test.go

```diff
@@ -340,28 +280,17 @@ func TestPRCheckout_differentRepo_remoteExists(t *testing.T) {
 	} } } }
 	`))
 
-	ranCommands := [][]string{}
-	//nolint:staticcheck // SA1019 TODO: rewrite to use run.Stub
-	restoreCmd := run.SetPrepareCmd(func(cmd *exec.Cmd) run.Runnable {
-		switch strings.Join(cmd.Args, " ") {
-		case "git show-ref --verify -- refs/heads/feature":
-			return &errorStub{"exit status: 1"}
-		default:
-			ranCommands = append(ranCommands, cmd.Args)
-			return &test.OutputStub{}
-		}
-	})
-	defer restoreCmd()
+	cs, cmdTeardown := run.Stub()
+	defer cmdTeardown(t)
+
+	cs.Register(`git fetch robot-fork \+refs/heads/feature:refs/remotes/robot-fork/feature`, 0, "")
+	cs.Register(`git show-ref --verify -- refs/heads/feature`, 0, "")
+	cs.Register(`git checkout feature`, 0, "")
+	cs.Register(`git merge --ff-only refs/remotes/robot-fork/feature`, 0, "")
 
 	output, err := runCommand(http, remotes, "master", `123`)
 	assert.NoError(t, err)
 	assert.Equal(t, "", output.String())
-
-	assert.Equal(t, 4, len(ranCommands))
-	assert.Equal(t, "git fetch robot-fork +refs/heads/feature:refs/remotes/robot-fork/feature", strings.Join(ranCommands[0], " "))
-	assert.Equal(t, "git checkout -b feature --no-track robot-fork/feature", strings.Join(ranCommands[1], " "))
-	assert.Equal(t, "git config branch.feature.remote robot-fork", strings.Join(ranCommands[2], " "))
-	assert.Equal(t, "git config branch.feature.merge refs/heads/feature", strings.Join(ranCommands[3], " "))
```

Comment:
This used to be stubbed, but is no longer present after the rewrite. Were the stubs outdated?

Reply:
No, a mistake on my part of changing test behavior by missing the non-zero exit code. This has been corrected.